src/components/DragSwitch/index.tsx

```diff
@@ -152,6 +152,7 @@ export const DragSwitch = ({
         }}
       />
       <input
+        className="switchInput"
```

Comment:
It looks like this component was not migrated to use `styled-components`. We shouldn't be using hard-coded class names any longer to avoid potential name collision.